catalog.html

```diff
@@ -0,0 +1,438 @@
+            </a>
+          </li>
+
+          <!-- Главная -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="page.html">Главная</a>
+          </li>
+
+          <!-- О Седоне -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="about_Sedona.html">О Седоне</a>
+          </li>
+
+          <!-- Гостиницы -->
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="hotel.html">Гостиницы</a>
+          </li>
+
+        </ul>
+        <ul class="navigation-list navigation-user">
+
+          <!-- Поиск -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+
+          <!-- В избранном -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">В избранном</span>
+              <span class="user-vaforites">12</span>
+            </a>
+          </li>
+
+          <!-- ХОЧУ СЮДА! -->
+          <li class="navigation-item">
+            <a class="button navigation-link" href="#">ХОЧУ СЮДА!</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+
+    <!-- Основная страница -->
+
+
+    <main class="main-inner" data-test="main">
+      <header class="inner-header">
+        <h1 class="inner-header-title">Гостиницы Седоны</h1>
+
+        <!-- Хлебные крошки -->
+        <ul class="breadcrumbs-list">
+
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link breadcrumbs-link-icon" href="index.html">
+              <svg width="13" height="15" viewBox="0 0 13 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M6.5 0.5L0.5 6.30006V12.5001H12.5V6.30006L6.5 0.5ZM11.5 11.5001H1.5V6.70005L6.5 1.9L11.5 6.70005V11.5001Z"
+                  fill="white" />
+                <path d="M4.5 10.5001H5.5V8.00005H7.5V10.5001H8.5V7.00005H4.5V10.5001Z" fill="white" />
+              </svg>
+              <svg width="8" height="21" viewBox="0 0 8 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M6.91526 9.96875L2.16526 5.71875C1.91526 5.46875 1.41526 5.46875 1.04026 5.71875C0.665263 5.96875 0.790263 6.46875 1.04026 6.71875L5.16526 10.4688L1.16526 14.2188C0.915263 14.4688 0.915263 14.9688 1.16526 15.2188C1.16526 15.3438 1.54026 15.4688 1.66526 15.4688C1.79026 15.4688 2.04026 15.3438 2.16526 15.2188L6.91526 10.9688C7.29026 10.7188 7.29026 10.2188 6.91526 9.96875Z"
+                  fill="white" />
+              </svg>
+              <span class="visually-hidden">Главная</span>
+            </a>
+          </li>
+
+          <li class="breadcrumbs-item breadcrumbs-item-text">
+            <a class="breadcrumbs-link breadcrumbs-link-text" href="index.html">Гостиницы</a>
+          </li>
```

Comment:
```suggestion
          <li class="breadcrumbs-item breadcrumbs-item-text">
            <a class="breadcrumbs-link breadcrumbs-link-text">Гостиницы</a>
          </li>
```

Последний пункт хлебных крошек это текущая страница. 
Таким образом, варианты:
1. Оставляем вообще просто текст внутри li
2. Меняем `a` на `span`
3.  Убираем href у `a`
4. Оставляем href и добавляем aria-current.

Мне наиболее симпатичными кажутся варианты 1 и 2. В меньшей мере 3.